neo/SmartContract/StateReader.cs

```diff
@@ -332,8 +355,19 @@ protected virtual bool Runtime_Deserialize(ExecutionEngine engine)
             using (MemoryStream ms = new MemoryStream(data, false))
             using (BinaryReader reader = new BinaryReader(ms))
             {
-                StackItem item = DeserializeStackItem(reader);
-                if (item == null) return false;
+                StackItem item;
+                try
+                {
+                    item = DeserializeStackItem(reader);
+                }
+                catch (FormatException)
```

Comment:
I prefer generic `catch`

Reply:
There is a generic catch at here, we can keep the specific exception for debugging.

https://github.com/neo-project/neo-vm/blob/b1338d488a4769938de648aaf7518a5f0c163afa/src/neo-vm/ExecutionEngine.cs#L957-L970